databuilder/query_language.py

```diff
@@ -423,3 +423,6 @@ class RoundToFirstOfMonth(ValueFromFunction):
 
 class RoundToFirstOfYear(ValueFromFunction):
     pass
+
+
+SelectedNode = Union[FilteredTable, RowFromAggregate, Row]
```

Comment:
These are the types that when we convert them we get a SQLAlchemy `Select` class -- the code that uses the `source` fields that have this type assumes that it can call that classes methods.

I'm not very happy with this. Maybe there is a `Node` type missing that could be a superclass of these? I think this points to a weakness in the query model that we can probably clean up later.